src/gauge.ts

```diff
@@ -231,3 +204,17 @@ export function handleRootGaugeRelativeWeightCapChanged(event: RelativeWeightCap
   gauge.relativeWeightCap = scaleDownBPT(event.params.new_relative_weight_cap);
   gauge.save();
 }
+
+export function handleRewardDurationUpdated(event: RewardDurationUpdated): void {
+  ChildChainRewardToken.create(event.params.reward_token);
```

Comment:
Will this not break when `handleRewardDurationUpdated` is emitted a second time for the same `reward_token`? I think we're used to only creating data sources from templates on events that are emitted only once (factory events) - do we need to add some sort of error handling here?